pom.xml

```diff
@@ -1206,7 +1206,6 @@
                 <module>samples/client/petstore/typescript-nestjs-v6-provided-in-root</module>-->
                 <!-- comment out due to error `npm run build`
                 <module>samples/client/petstore/typescript-jquery/npm</module>-->
-                <module>samples/client/petstore/typescript-angular-v11-provided-in-root</module>
```

Comment:
Please copy these unit tests to the newest amgular version, otherwise we dont have any angular typescript unit tests

Reply:
@macjohnny Thanks for your review.
OK. I've just done that at 79484ce (Migration was done with reference to https://update.angular.io/)